index.js

```diff
@@ -1563,7 +1563,8 @@
   //. Nothing
   //. ```
   function reject(pred, filterable) {
-    return filter(function(x) { return !pred(x); }, filterable);
+    return Filterable.methods
+      .filter(filterable)(function(x) { return !pred(x); });
```

Comment:
I would find this approach to line wrapping slightly more pleasing:

```javascript
var filter = Filterable.methods.filter(filterable);
return filter(function(x) { return !pred(x); });
```

Thoughts?

Reply:
It does look nicer and a compiler should optimize that away. Right?

Reply:
Perhaps:
```js
var filter = Filterable.methods.filter;
return filter(filterable)(function(x) { return !pred(x); });
```
This way the name matches the operation.

Reply:
I like it!

Reply:
Updated